feat: build process and publishing for web-pkg (#10799)
owncloud/web: packages/web-pkg/README.md

# web-pkg

> `web-pkg` is a package that provides utilities that can be useful when creating and developing extensions for ownCloud Web.

Reviewer: ```suggestion
`web-pkg` is a package that provides utilities, most importantly a variety of components and composables, that can be useful when developing apps and extensions for ownCloud Web.
```

> It's important to note that this package solely provides types and lacks an actual standalone implementation. This is because `web-pkg` is intended solely for Web extension development, and the real implementation is already provided by the Web runtime.

Reviewer: ```suggestion
It's important to note that this package solely provides types because `web-pkg` is intended solely for Web extension development. The actual implementation of the utilities is provided by the Web runtime automatically.
```
